pulp_ansible/app/migrations/0032_collectionversion_execution_environment.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.18 on 2021-02-08 20:36
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ansible', '0030_collectionversion_execution_environment'),
```

Comment:
it depends on the 31
```suggestion
        ('ansible', '0031_ansiblerepository_last_synced_metadata_time'),
```

Reply:
:facepalm: